clients/client-java/src/main/java/org/apache/gravitino/client/GravitinoClient.java

```diff
@@ -242,6 +242,7 @@ public boolean deleteRole(String role) throws NoSuchMetalakeException {
    * @return The created Role instance.
    * @throws RoleAlreadyExistsException If a Role with the same name already exists.
    * @throws NoSuchMetalakeException If the Metalake with the given name does not exist.
+   * @throws NoSuchMetadataObjectException If securable object doesn't exist
    * @throws RuntimeException If creating the Role encounters storage issues.
    */
   public Role createRole(
```

Comment:
Should you also add this exception to the interface signature?

Reply:
Added.